staging/src/k8s.io/apiserver/pkg/util/proxy/dial_test.go

```diff
@@ -96,7 +96,7 @@ func TestDialURL(t *testing.T) {
 			ts.TLS = &tls.Config{Certificates: []tls.Certificate{cert}}
 			ts.StartTLS()
 
-			tlsConfigCopy := utilnet.CloneTLSConfig(tc.TLSConfig)
+			tlsConfigCopy := tc.TLSConfig
```

Comment:
Looks fine. Lgtm to me if https://github.com/kubernetes/kubernetes/pull/41758/files/1406c7a485d6edf793740a66eb137a5afaa9e259#r110609367 is readded.